evaluate/task/test-integration/task_test.go

```diff
@@ -93,4 +100,85 @@ func TestWriteTestsRun(t *testing.T) {
 			assert.Contains(t, data, "msg=\"evaluated model\" model=symflower/symbolic-execution")
 		},
 	})
+	{
+		mockProvider := providertesting.NewMockQuery(t)
+		model := llm.NewModel(mockProvider, "model")
+		validate(t, &tasktesting.TestCaseTask{
+			Name: "Rust",
+
+			Setup: func(t *testing.T) {
+				mockProvider.On("Query", mock.Anything, mock.Anything, mock.Anything, mock.Anything).Return(
+					&provider.QueryResult{
+						Message: bytesutil.StringTrimIndentations(`
+							` + "```rust`" + `
+							#[cfg(test)]
+							mod tests {
+								use super::*;
+
+								#[test]
+								fn test_plain() {
+									plain();
+								}
+							}
+							` + "```" + `
+						`),
+					},
+					nil,
+				).After(100 * time.Millisecond)
+			},
+
+			Model:          model,
+			Language:       &rust.Language{},
+			TestDataPath:   filepath.Join("..", "..", "..", "testdata"),
```

Comment:
I guess the changes in this test file would need to go in the commit where we add the repository.